### ISSUE TITLE
[MAJOR BUG] Impossible get the public key and any other package for prebuilt-mpr of apt repository because now the all APT URLs requires login

### ISSUE THREAD

**Issue by luzfcb** [makedeb/prebuilt-mpr#163]
Something was changed, and now all URLs related to the APT repository require login.

```bash
luzfcb@luzfcb-pc:~$ sudo pkcon refresh
Refreshing cache                        [=========================]         
Loading cache                           [=========================]         
Downloading update information          [=========================]         
Running                                 [=========================]         
Finished                                [=========================]         
Fatal error: E: Failed to fetch https://proget.makedeb.org/dists/prebuilt-mpr/InRelease  Clearsigned file isn't valid, got 'NOSPLIT' (does the network require authentication?)
E: The repository 'https://proget.makedeb.org prebuilt-mpr InRelease' is no longer signed.
W: Updating from such a repository can't be done securely, and is therefore disabled by default.
W: See apt-secure(8) manpage for repository creation and user configuration details.
E: https://proget.makedeb.org prebuilt-mpr InRelease is not (yet) available (Clearsigned file isn't valid, got 'NOSPLIT' (does the network require authentication?))
```

from official documentation: https://docs.makedeb.org/prebuilt-mpr/getting-started/#setting-up-the-repository

```bash
luzfcb@luzfcb-pc:~$  wget -qO - 'https://proget.makedeb.org/debian-feeds/prebuilt-mpr.pub' | gpg --dearmor | sudo tee /usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg 1> /dev/null
gpg: no valid OpenPGP data found.

```

If I try to get the public key directly by accessing the URL https://proget.makedeb.org/debian-feeds/prebuilt-mpr.pub, or access the InRelease https://proget.makedeb.org/dists/prebuilt-mpr/InRelease, I get this page:

![image](https://github.com/makedeb/prebuilt-mpr/assets/807599/3d5d9a28-27f4-4710-aab7-503266acb676)

The https://proget.makedeb.org/ URL shows something related to missing license 

![image](https://github.com/makedeb/prebuilt-mpr/assets/807599/cfcc0645-6983-4665-8d6c-c8724036be90)

**Comment by hwittenborn** [comment]
Thanks for the heads up @luzfcb! Looks like there was something janky going on with the licensing (I'm not sure why, I was on a free license plan that never expired), but I went ahead and got a new license key and everything looks good on my end. Can you see if it's the same for you?

**Comment by luzfcb** [comment]
@hwittenborn Thanks for the quick response. Everything is now working again as expected.

I had to reinstall the public key, but that might have been a side effect of trying to install the key when there was a problem.

After reinstalling the key, everything was back to working as expected.


For anyone that has this error message:

```bash
luzfcb@luzfcb-pc:~$  sudo pkcon refresh
Refreshing cache                        [=========================]         
Loading cache                           [=========================]         
Downloading update information          [=========================]         
Running                                 [=========================]         
Finished                                [=========================]         
Fatal error: W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://proget.makedeb.org prebuilt-mpr Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 4FE9F2C43D9428A0
E: https://proget.makedeb.org prebuilt-mpr Release.gpg is not (yet) available (The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 4FE9F2C43D9428A0)

```

Try to fix by force reinstall the public key via:

```bash
wget -qO - 'https://proget.makedeb.org/debian-feeds/prebuilt-mpr.pub' | gpg --dearmor | sudo tee /usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg 1> /dev/null
```
